docs/content/tutorials/tutorial-compaction.md

```diff
@@ -31,7 +31,7 @@ There will be 24 segments for this datasource, one segment per hour in the input
 
 Running a COUNT(*) query on this datasource shows that there are 39,244 rows:
 
-```
+```bash
```

Comment:
hm, it's not purely SQL, and it shows the output from a shell, so I'll keep this as bash for now

Reply:
Okay, this also makes sense. :sweat_smile:

Reply:
yeah, I think there's some ambiguity in how to classify it, in the end I decided to go with "bash" because I can say that all parts of that snippet are "shell text" but I can't say that all parts are "sql".

Reply:
Indeed, thanks for explaining this.